src/charactersheet/components/mini-plus-minus.js

```diff
@@ -43,17 +47,19 @@ ko.components.register('mini-plus-minus', {
     viewModel: MiniPlusMinusComponentViewModel,
     template: '\
     <div class="btn-group btn-group-vertical" role="group">\
-        <!-- ko if: !hideDown -->\
```

Comment:
I updated this in v2 to change the look and feel of it. Updated with new `enabled` flag usage.